lib/rubocop/cop/style/space_around_block_parameters.rb

```diff
@@ -14,17 +14,26 @@ module Style
       #   {}.each { |x, y| puts x }
       class SpaceAroundBlockParameters < Cop
         include ConfigurableEnforcedStyle
+        BEFORE_FIRST = 'Space before first'
+        AFTER_LAST = 'Space after last'
+        BEFORE_BLOCK = 'before first block parameter'
+        AFTER_BLOCK = 'after last block parameter'
+        EXTRA_BEFORE = 'Extra space before'
+        EXTRA_BEFORE_FIRST = 'Extra space before first'
+        EXTRA_AFTER = 'Extra space after last'
+        AFTER_CLOSING_PIPE = 'after closing `|`'
```

Comment:
Splitting mass assignment into separate assignments caused a method too long error on `check_inside_pipes`. I extracted the strings into constants so that I could fit some of the statements on one line and shorten the length of the method. Alternatively, I could extract each condition into their own method. 

I can extract this change out into a different commit if need be.